core/src/main/java/hudson/model/Computer.java

```diff
@@ -199,7 +199,7 @@
      * @see Executor#resetWorkUnit(String)
      */
     private transient final List<TerminationRequest> terminatedBy = Collections.synchronizedList(new ArrayList
-            <TerminationRequest>());
+            <>());
```

Comment:
Seems like this line could be collapsed now?

Reply:
I'm not really pretending to speak Java at this point. There are more commits in my queue:
https://github.com/jsoref/jenkins/tree/java-cleanup-full

But, these are (generally) things that IntelliJ is suggesting.

Reply:
I mean you can combine the two lines now that it's not so long. More of a style thing, so if you'd prefer to do that in a separate PR, that's fine, too.

Reply:
Oh, heh. I'll see how smart GitHub diff is. If it's reasonable then I might amend this, if not, I'll add an extra commit.

Reply:
I checked, it doesn't do a good job of showing the change, so it'll have to be a new commit.

Reply:
Fine with me. A PR for cleaning up formatting makes sense separate from code refactoring.